services/web-app/pages/developing/index.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Developing
+description:
+---
+
+<PageDescription>
+  Welcome to Carbon! If you’re just started out developing with Carbon, here’s a checklist of
+  everything you need to get up and running.
+</PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>Step 1: Learn about Carbon</AnchorLink>
+  <AnchorLink>Step 2: Get started</AnchorLink>
+  <AnchorLink>Step 3: Explore developer tools</AnchorLink>
+</AnchorLinks>
+
+## Step 1: Learn about Carbon
+
+Carbon translates the [IBM Design Language](http://www.ibm.com/design/language) into resources for
+building digital experiences. Take the time to read the site so that you fully understand what
+drives IBM’s design philosophy and principles, and can make informed decisions in your product work.
+
+By adopting Carbon, front-end developers and engineers gain a collection of reusable components to
+build websites and user interfaces quickly and efficiently.
+
+### Understand the ecosystem
+
+The following diagram shows the relationships of assets and guidance across IBM. The Carbon Design
+System is often extended or complimented by additional product and digital systems.
+
+## Step 2: Get Started
```

Comment:
is this the correct spacing (h2 to h3)? looks a bit different to me but couldn't find exact measurements on the figma page 🤔 
<img width="584" alt="image" src="https://user-images.githubusercontent.com/40550942/171468688-0e33190c-bca2-42a5-90fb-4df116cdc5d2.png">
<img width="709" alt="image" src="https://user-images.githubusercontent.com/40550942/171468727-bf6c5a40-8969-4201-ad67-5fd7cfce5942.png">

Reply:
The spacing matches the current website / Gatsby theme, so I think we're good. I think the sketch file is off. OR if design wants a global change we can make a new issue to adjust any spacing site-wide.